docs/data/duplication.md

```diff
@@ -33,19 +33,19 @@ The same content may be served on separate domains on the web. For example, some
 
 As described in [IDs and URls](ids-and-urls), if a webpage provides a Canonical URL we will use it to refer to that webpage. We also include the `subj.url`, which is the URL that was actually visited. The Newsfeed Agent may visit the same blogpost more than once, but if it does this you you filter out repeated observations of the same links by ignoring duplicate `subj_id`, `obj_id` pairs.
 
-### DOIs vs Landing Pages
+### DOIs vs landing pages
 
 The Crossref Agents attempt to convert every link or mention of an item to an Event. On some pages, for example Wikipedia, the same reference may be linked several different ways. An Agent treats each of these links independently, which means that you may see more than one link between a given Wikipedia page and an article by their `subj_id` field, but the `subj.url` will indicate different URLs.
 
-How you interpret these is up to you. For some use-cases, it is important to be able to tell the difference between links via an Article Landing Page and the DOI. In some cases, for example component DOIs or DOIs that represent versioned content, the DOI could more specific than the Article Landing Page. In other cases, it's not so important, so duplicates can be removed based on the `subj_id` and `obj_id`.
+How you interpret these is up to you. For some use-cases, it is important to be able to tell the difference between links via an article landing Page and the DOI. In some cases, for example component DOIs or DOIs that represent versioned content, the DOI could more specific than the article landing page. In other cases, it's not so important, so duplicates can be removed based on the `subj_id` and `obj_id`.
 
-## Duplication as Corroboration
+## Duplication as corroboration
 
 Every source has different characteristics which must be taken into account when deciding how to interpret the data, including classification and mitigation of duplicates. However, you may find that between sources, the presence of a duplicate indicates something. For example, if two entirely different sources make the same assertion that links two things, and they are working from different data, you might wish to treat that duplication as a corroboration of the assertion. 
 
-## Repeated Observations
+## Repeated observations
 
-The Web source Agent may check the same webpage more than once over time. In this case it may make the same observations repeatedly. If you find an Event that has a duplicate, or is very similar to another Event made a while ago, this may be the reason.
+The web source Agent may check the same webpage more than once over time. In this case it may make the same observations repeatedly. If you find an Event that has a duplicate, or is very similar to another Event made a while ago, this may be the reason.
```

Comment:
This is unclear if not capitalized. 'Web' in title case refers to the specific Agent, not the WWW.